core/src/main/java/forklift/source/decorators/Queues.java

```diff
@@ -1,14 +1,17 @@
-package forklift.decorators;
+package forklift.source.decorators;
```

Comment:
Yeah. The `forklift.decorators` namespace is getting cluttered and the source annotations have a substantially different meaning from the rest of the forklift annotations.

It's obviously not a necessary change, but I think having good code organization is important when you can get it for cheap.